From 386fa849157dabde42ab39de7acb31750956490c Mon Sep 17 00:00:00 2001
From: Adam Duskett <aduskett@gmail.com>
Date: Mon, 3 Feb 2020 06:21:57 -0800
Subject: [PATCH] set minimum cmake version to 3.10

The CMakeLists.txt file uses the CMP0091 which is an MSVC runtime library flag
abstraction macro.

Because we are not building ninja for Microsoft, it is safe to remove this
macro and set the minimum version to 3.10.

Signed-off-by: Adam Duskett <aduskett@gmail.com>
[Rebased for ninja-1.10.2 - CMP0091 usage was removed upstream by
https://github.com/ninja-build/ninja/commit/e6f5bcffb3c561d228d88ff26edab26575b7d520]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7f03c35..4b61479 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.10)
 
 include(CheckIncludeFileCXX)
 include(CheckIPOSupported)
-- 
2.29.2

